python/ray/data/_internal/execution/util.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+import ray
+from ray.data.block import Block, BlockAccessor
+from ray.data._internal.execution.interfaces import RefBundle
+
+
+def _make_ref_bundles(simple_data: List[Block]) -> List[RefBundle]:
```

Comment:
shall we add comments for these utility methods, as they might be used across multiple places?

Also why are they named with leading `_`?
`_make_ref_bundles` -> `make_ref_bundles`?